### DISCUSSION TITLE
Compact view for list is absolutely needed

### DISCUSSION THREAD

**Discussion by deemytch** [community/community#50097] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For a quick overview on own list, also for a quick search with own eyes an compact representation as a table I think is absolutely good.
_For example a table with just tree columns and font size at 9pt_ 
```
author  | name                          | language | last update
dwyl    | elixir-auth-microsoft         | Elixir   | 2 days ago

```

**Comment by blanchekorn55** [comment]
Nothing anders to nobody

**Reply by blanchekorn55** [reply]
Me know

**Reply by blanchekorn55** [reply]
Use the first one

**Reply by Facebook979** [reply]
I am a freelancer ztp beginner and GTP marketing with certified great learning Academy but this site is newly for me

**Reply by Me1ad1** [reply]
🚀👏👏

**Comment by 26e43ee** [comment]
- [x] orange money
